public/app/core/utils/accessControl.ts

```diff
@@ -0,0 +1,9 @@
+import config from '../../core/config';
+
+// addAccessControlQueryParam appends ?accesscontrol=true to a url when accesscontrol is enabled
+export function addAccessControlQueryParam(url: string): string {
```

Comment:
@mckn same, followed your suggestion to create a utility function, but would you have placed it somewhere else?

Reply:
I this that is good enough for now :)